client/src/components/Toasts/ToastActions.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'components/Button/Button';
+
+/**
+ * Display an individual toast action. Actions can be button or links.
+ * When an href prop is provided, we assume the action is meant to be a link.
+ */
+const ToastAction = ({ label, href, onClick, dismissed, onDismiss }) => {
+  const props = href ?
+    { href, tag: 'a' } :
+    { onClick: (e) => {
+      e.preventDefault();
+
+      // When the toast is in the process of being dismiss, suppress action
+      if (!dismissed) {
+        if (onClick) { onClick(); }
+        // Dismiss the toast after it has fired its action
+        onDismiss();
+      }
+    } };
+
+  return <Button color="link" className="toast__action" {...props}>{label}</Button>;
+};
+
+/**
+ * Display a list of toast actions. Up to 2 actions can be displayed.
```

Comment:
Sorry if you've already covered this, but why 2?

Reply:
That's what the designs call for. I didn't pick Paul's brain on why two. My guess is that it's a size constraint.